src/libutil/file-descriptor.hh

```diff
@@ -106,15 +106,42 @@ void drainFD(
 #endif
     );
 
+/**
+ * Get [Standard Input](https://en.wikipedia.org/wiki/Standard_streams#Standard_input_(stdin))
+ */
+[[gnu::always_inline]]
```

Comment:
Do we need the always_inline? Might be a bit of a micro-optimisation.

Reply:
I don't know. @puffnfresh did this, so I defer to him.

Reply:
(Gonna merge in the meantime since this decision is orthogonal)